.codedeploy/scripts/add_node_specific_config

```diff
@@ -23,10 +23,10 @@ FLAGS=
 
 case "$NAME" in
 	vln)
-		FLAGS="--ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all --node-key $VLN1_KEY"
+		FLAGS="--collator --ws-port 80 --rpc-port 8080 --rpc-external --ws-external --rpc-cors all -- --execution wasm --chain rococo --port 30336"
 		NAME="node"
 		;;
-	*) FLAGS="--validator $(bootnodes vln)";;
+	*) FLAGS="";;
```

Comment:
https://tenor.com/view/parks-and-rec-chris-pratt-no-idea-what-iam-doing-confused-gif-4679222

Reply:
Ok, if I understand correctly the update in https://github.com/valibre-org/vln-node/pull/80/commits/6a6cbc9e3d07a72bf787d95a43c44d4fa53e4452 should start all 3 as collators